kvmagent/kvmagent/plugins/host_model_mount_plugin.py

```
@@ -201,6 +450,13 @@ def mount_model_center(self, req):
             # Mount
             success, error = mount_juicefs(cmd.storageUrl, mount_point)
             if success:
+                try:
+                    _save_mount_registry_entry(cmd.modelCenterUuid, cmd.storageUrl, mount_point)
+                except Exception as save_err:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)